classes.js

```diff
@@ -0,0 +1,61 @@
+// 1. Copy and paste your prototype in here and refactor into class syntax.
+
+// Parent Class:
+
+class CuboidMaker {
+    constructor(obj) {
+        this.length = obj.length;
+        this.width = obj.width;
+        this.height = obj.height;
+    }
+
+    volume() {
+        const v = this.length * this.width * this.height;
+        return v;
+    }
+
+    surfaceArea() {
+        const sa = 2 * (this.length * this.width + this.length * this.height + this.width * this.height);
+        return sa;
+    }
+
```

Comment:
Mvp requires that you are also using 'new' make sure you go back to the instructions and read them over so you can complete that. Otherwise your classes are looking great